app/src/main/java/id/rizmaulana/covid19/ui/dailygraph/DailyDataListFragment.kt

```diff
@@ -0,0 +1,62 @@
+package id.rizmaulana.covid19.ui.dailygraph
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.recyclerview.widget.LinearLayoutManager
+import id.rizmaulana.covid19.databinding.FragmentDailyListBinding
+import id.rizmaulana.covid19.ui.adapter.DailyFactoryImpl
+import id.rizmaulana.covid19.ui.adapter.VisitableRecyclerAdapter
+import id.rizmaulana.covid19.ui.base.BaseFragment
+import id.rizmaulana.covid19.util.ext.observe
+import org.koin.android.viewmodel.ext.android.sharedViewModel
+
+class DailyDataListFragment : BaseFragment() {
+
+    private val viewModel by sharedViewModel<DailyGraphViewModel>()
+
+    private var _binding: FragmentDailyListBinding? = null
```

Comment:
why not make it `lateinit var` instead?

Reply:
oh that's actually simpler, great, will do